### ISSUE TITLE
What does this project do?

### ISSUE THREAD

**Issue by redfast00** [ZeusWPI/ZeusWPI#4]
What does this project do? Is it still active?

**Comment by werthen** [comment]
Yes it's very much still actively used. It's running on https://zeus.ugent.be/zeuswpi/ ask us for the credentials
